src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,51 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final String LOGIN_EXISTS = "user with such login already exists";
+    private static final String LOGIN_IS_NULL = "user's login is null";
+    private static final String USER_IS_NULL = "user is null";
+    private static final String PASSWORD_IS_NULL = "user's password is null";
+    private static final String TOO_YOUNG = "user must first grow up a little; age less than 18";
+    private static final String NEGATIVE_AGE = "negative user's age";
+    private static final String LOGIN_HAS_INVALID_SYMBOL = "login has at least one invalid symbol";
+    private static final String PASSWORD_HAS_INVALID_SYMBOL
+            = "Password has at least one invalid symbol";
+    private static final String LOGIN_HAS_LESS_THAT_SIX_SYMBOLS
+            = "login too short, must be at least six symbols";
```

Comment:
It's actually redundant to use constants for exception messages.
It just gets harder to work with it. You have to scroll to the top of the class to see the message while reading the code and it's almost impossible for you to reuse it.